.github/workflows/workspace.yaml

```diff
@@ -1,37 +1,37 @@
 name: Workspace
 on:
-  pull_request:
-    paths-ignore:
-      - README.md
-
+  push: 
+    branches: [ "feature/what" ]
```

Comment:
```suggestion
  pull_request:
  push: 
    branches:
      - main
      - develop
      - feature/*
```

Reply:
should be fixed now, note that I omitted the `push:`